src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly make the value of the  action type to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:  `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`
*